test/functional/rpc_psbt.py

```diff
@@ -834,6 +847,9 @@ def test_psbt_input_keys(psbt_input, keys):
         assert_raises_rpc_error(-8, "PSBTs not compatible (different transactions)", self.nodes[0].combinepsbt, [psbt1, psbt2])
         assert_equal(self.nodes[0].combinepsbt([psbt1, psbt1]), psbt1)
 
+        if self.options.descriptors:
+            self.log.info("Test that PSBT_OUT_TAPROOT_TREE is only present for Taproot outputs that have script paths")
+
```

Comment:
Looks like you actually didn't commit this test?

Reply:
Removed. I changed how I wanted to do the test and forgot to drop the log line.